lib/internal/util/inspect.js

```diff
@@ -1999,21 +1999,18 @@ function formatProperty(ctx, value, recurseTimes, key, type, desc,
       SymbolPrototypeToString(key),
       escapeFn,
     );
-    name = `[${ctx.stylize(tmp, 'symbol')}]`;
-  } else if (key === '__proto__') {
-    name = "['__proto__']";
-  } else if (desc.enumerable === false) {
-    const tmp = RegExpPrototypeSymbolReplace(
-      strEscapeSequencesReplacer,
-      key,
-      escapeFn,
-    );
-    name = `[${tmp}]`;
+    name = ctx.stylize(tmp, 'symbol');
   } else if (RegExpPrototypeExec(keyStrRegExp, key) !== null) {
     name = ctx.stylize(key, 'name');
   } else {
     name = ctx.stylize(strEscape(key), 'string');
   }
+
+  if (key === '__proto__') {
```

Comment:
Can you please move this if inside of https://github.com/nodejs/node/pull/55778/files#diff-ee826b416a72014ac81621cf3ecaeec6adcc40163f60775d157aa94618c80201R2003?

That way it's a single if less for other cases.

Reply:
sure - although then it would still hit the enumerability "if", which would mean if it's a non-enumerable `__proto__` data property it'd get double brackets?

Reply:
we decided to go with the double brackets